src/components/ActivityGrid.vue

```diff
@@ -38,7 +38,7 @@
 			<tr v-for="(activity, activityKey) in group.activities" :key="activityKey">
 				<th scope="row">
 					<!-- eslint-disable vue/no-v-html -->
-					<label @click="toggleMethodsForActivity({groupKey, activityKey})" v-html="activity.desc" />
+					<label v-html="activity.desc" />
```

Comment:
This method was used only here, so we can also remove it from the component's methods (`mapActions`, in line 85).

But keeping the method in the store, if case we add a "check all" button later.

Reply:
Should we make this element just a `span`/`div`, as this `<label>` has no linked elements?

Reply:
Or shall we link it to the checkbox ?

Reply:
> Or shall we link it to the checkbox ?

We cannot, there are 2 checkboxes